rust/README.md

```diff
@@ -0,0 +1,42 @@
+# Rust on CodeQL
+
+## Development
+
+### Dependencies
+
+At present you need to be in the `rust-experiment` branch (or a branch on top of that) in both the `semmle-code` and `codeql` repos.
```

Comment:
I'd scratch this: in any case this README is on this branch, so this information seems redundant. And once we merge to `main`, we've got nothing to update if this line is not present :slightly_smiling_face:

Reply:
True, but I was hit by the gotcha of having only checked out the right branch on `codeql` this morning.

When do we plan to merge to `main`?

Reply:
I've removed the note.